pyflow/graphics/view.py

```diff
@@ -447,6 +450,14 @@ def get_block_below_mouse(
 
     def drag_edge(self, event: QMouseEvent, action="press"):
         """Create an edge by drag and drop."""
+
+        # edge creation / destruction if control is pressed
+        if event is None or (
+            action != "move"
+            and QApplication.keyboardModifiers() == Qt.KeyboardModifier.ControlModifier
+        ):
+            return event
```

Comment:
```suggestion
        # edge creation / destruction if control is pressed
        ctrl_pressed = QApplication.keyboardModifiers() == Qt.KeyboardModifier.ControlModifier
        if event is None or (action != "move" and ctrl_pressed):
            return event
```
Use variable name for long logic control